src/com/github/mtdrewski/GRAPH_moment/controller/GraphDrawerController.java

```diff
@@ -29,6 +31,7 @@
     private boolean isUnsaved = false;
 
     private enum Mode {EDGE, SELECT, STANDARD}
+    protected boolean isDirected = false;
 
```

Comment:
shouldn't this information be in the graph.java? I think that there could be a moment where we would need this information from graph, but not from controller.

Reply:
There are separate classes Graph and DirectedGraph, and storing a boolean inside one or each of them seems not elegant for me. I managed to make everything work without need of getting this info out of graph.java